docker-compose.yml

```diff
@@ -62,7 +62,8 @@ services:
   mysql:
     container_name: mysql
     # https://hub.docker.com/r/library/mysql/
-    image: mysql:5.7.29
+    image: mysql:8.0.19
+    command: [--default-authentication-plugin=mysql_native_password]
```

Comment:
Could you comment or point us to the documentation as to why this is needed?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html
The ```important``` section tells us this